token-lending/program/src/state/reserve.rs

```diff
@@ -489,7 +489,7 @@ pub struct ReserveCollateral {
     /// Reserve collateral mint address
     pub mint_pubkey: Pubkey,
     /// Reserve collateral mint supply, used for exchange rate
-    pub mint_total_supply: u64,
+    pub mint_amount: u64,
```

Comment:
@jstarry Let me know what you think if this change, I can revert if it's confusing. I think using `supply` to represent an amount/quantity when it's used similarly for pubkeys makes it harder to reason about.

Reply:
We typically use "total supply" to describe this number in RPC so it seems more intuitive to treat it as a number. I think the "supply" pubkey naming should change instead.

Reply:
Good point. Do you have an idea for that?

Reply:
not yet, it would be nice to do a follow up with renames across the board for the token accounts though

Reply:
Reverted for now, but we can revisit later.